### ISSUE TITLE
[Suggestion] A shard wand for survival

### ISSUE THREAD

**Issue by kolhed** [McJty/RFTools#1287]
A wand that we can infuse, and then use the wand to infuse a block without having to break it and infuse it.
Maybe slower to infuse wand then block, maybe more expensive.. If needed for balancing

**Comment by McJty** [comment]
Yes, I have also wanted to have this. Never got around to implementing it yet though :-)

**Comment by Umber0010** [comment]
apologies for the necro post, but does that mean I could get an infused infuser without having to craft a second infuser to infuse the first infuser?
